### PR TITLE
sus

### PR DESCRIPTION
**Pull request by torontofangirl**
idk

### CODE REVIEW

File: Loungeware/Loungeware.yyp [hunk 1]
```diff
@@ -159,7 +161,6 @@
     {"id":{"name":"baku_mine_par_block","path":"objects/baku_mine_par_block/baku_mine_par_block.yy",},"order":9,},
     {"id":{"name":"___advanced_text","path":"scripts/___advanced_text/___advanced_text.yy",},"order":8,},
     {"id":{"name":"mimpy_firealarm_rm","path":"rooms/mimpy_firealarm_rm/mimpy_firealarm_rm.yy",},"order":1,},
-    {"id":{"name":"baku_mine_spr_pick","path":"sprites/baku_mine_spr_pick/baku_mine_spr_pick.yy",},"order":14,},
```
Comment (bakumoe) [inline Loungeware/Loungeware.yyp:162]: excuse me?!

File: Loungeware/Loungeware.yyp [hunk 2]
```diff
@@ -309,16 +312,17 @@
     {"id":{"name":"space_lander_snd_thrust","path":"sounds/space_lander_snd_thrust/space_lander_snd_thrust.yy",},"order":4,},
     {"id":{"name":"space_lander_sng_zandy_aliens_edit","path":"sounds/space_lander_sng_zandy_aliens_edit/space_lander_sng_zandy_aliens_edit.yy",},"order":5,},
     {"id":{"name":"___snd_cart_insert","path":"sounds/___snd_cart_insert/___snd_cart_insert.yy",},"order":0,},
+    {"id":{"name":"tfg_swipe_spr_spacefloosh","path":"sprites/tfg_swipe_spr_spacefloosh/tfg_swipe_spr_spacefloosh.yy",},"order":11,},
     {"id":{"name":"baku_mine_obj_block_diamond","path":"objects/baku_mine_obj_block_diamond/baku_mine_obj_block_diamond.yy",},"order":0,},
     {"id":{"name":"space_scooter_spr_dpad","path":"sprites/space_scooter_spr_dpad/space_scooter_spr_dpad.yy",},"order":13,},
     {"id":{"name":"___snd_cart_remove","path":"sounds/___snd_cart_remove/___snd_cart_remove.yy",},"order":1,},
     {"id":{"name":"n8fl_escape1_car","path":"objects/n8fl_escape1_car/n8fl_escape1_car.yy",},"order":9,},
-    {"id":{"name":"baku_mine_rm","path":"rooms/baku_mine_rm/baku_mine_rm.yy",},"order":6,},
```
Comment (bakumoe) [inline Loungeware/Loungeware.yyp:316]: so rude :<

File: Loungeware/Loungeware.yyp [hunk 3]
```diff
@@ -668,7 +681,7 @@
     {"roomId":{"name":"anti_hats_rm_start","path":"rooms/anti_hats_rm_start/anti_hats_rm_start.yy",},},
     {"roomId":{"name":"tfg_swipe_rm_game","path":"rooms/tfg_swipe_rm_game/tfg_swipe_rm_game.yy",},},
     {"roomId":{"name":"___rm_pause","path":"rooms/___rm_pause/___rm_pause.yy",},},
-    {"roomId":{"name":"baku_mine_rm","path":"rooms/baku_mine_rm/baku_mine_rm.yy",},},
```
Comment (bakumoe) [inline Loungeware/Loungeware.yyp:671]: do you have a personal vendetta against baku_mine ;_;

**Review by net8floz — CHANGES_REQUESTED** [review]
Be nicer to baku

**Review by NuxiiGit — COMMENTED** [review]
why `.yy.new` files?

File: Loungeware/Loungeware.yyp [hunk 4]
```diff
@@ -669,6 +684,7 @@
     {"roomId":{"name":"tfg_swipe_rm_game","path":"rooms/tfg_swipe_rm_game/tfg_swipe_rm_game.yy",},},
     {"roomId":{"name":"___rm_pause","path":"rooms/___rm_pause/___rm_pause.yy",},},
     {"roomId":{"name":"baku_mine_rm","path":"rooms/baku_mine_rm/baku_mine_rm.yy",},},
+    {"roomId":{"name":"","path":"",},},
```
Comment (net8floz) [inline Loungeware/Loungeware.yyp:687]: kinda sus tho

**Review by net8floz — CHANGES_REQUESTED** [review]

File: Loungeware/Loungeware.yyp (hunk 4 above)
Comment (NuxiiGit) [reply Loungeware/Loungeware.yyp:687]: delete

**Comment by NuxiiGit** [comment]
moved to #80

**Pull request closed without merging**